src/util/lockRetryOptions.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// docs: https://github.com/moxystudio/node-proper-lockfile
+
+export const lockOptions = { stale: 10_000 };
```

Comment:
if something exits unexpectedly, the lock is considered stale after 10s to avoid the "my files are locked and I can't do anything" bug.